islandora_solr.install

```diff
@@ -328,7 +330,6 @@ function islandora_solr_search_settings_variables() {
     'islandora_solr_primary_display',
     'islandora_solr_request_handler',
     'islandora_solr_luke_timeout',
-    'islandora_solr_breadcrumbs_parent_fields',
```

Comment:
Documentation actually indicates that it shouldn't be down there, as it's part of the islandora_solr_admin_breadcrumbs_settings form, whereas those variables are associated with the islandora_solr_admin_settings form.

Problem is [over here](https://github.com/Islandora/islandora_solr_search/blob/f434a5db068dc612c092cbc3182780f143de9c1b/includes/admin.inc#L786-L793); if someone hits the 'reset' button on the Solr settings form, it'll also inadvertently reset the breadcrumb parent fields on a completely separate form